packages/analytics-js-integrations/src/integrations/Clevertap/browser.js

```diff
@@ -67,6 +67,7 @@ class Clevertap {
       onUserLogin: [],
       notifications: [],
       privacy: [],
+      region: [],
```

Comment:
Incorrect.
As per their documentation, it's not an array and we have to define a default value.
https://developer.clevertap.com/docs/web-quickstart-guide#add-region-code

Reply:
addressed, kept default region value as none

Reply:
We should not define any default value if the region is not explicitly provided by the user as per their docs. Sorry, my bad.